backend/src/backend/main.py

```diff
@@ -137,4 +137,18 @@ async def add_book(book_payload: BookAddSchema):
     return {"msg": "ok", "uuid": str(book.id), "title": book.title.ident}
 
 
+@database.transaction()
+@api.get("/titles/", status_code=200, response_model=TitlesListSendSchema)
+async def receive_titles():
```

Comment:
I don't understand what `receive_titles` mean. Let's use the following convention everywhere:

* `list_titles`
* `get_title`
* `create_title`
* `update_title`
* `delete_title`
* `delete_titles` (we won't have that I believe)